### PR TITLE
Doc changes for FASC v2.7

### PR DESCRIPTION
**Pull request by manuwelakanade-GSLAB**
Doc changes for FASC v2.7

### CODE REVIEW

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : Can you please review and merge this pull request. Thanks.

File: docs/faq.rst
Comment (JosephPJordan) [inline docs/faq.rst:65]: ... when performing a BIG-IP login..." is better.

File: docs/faq.rst
Comment (JosephPJordan) [inline docs/faq.rst:67]: ...BIG-IP has changed, and you attempt to re-login to the BIG-IP from the FASC app, you may see this error.

File: docs/faq.rst
Comment (JosephPJordan) [inline docs/faq.rst:69]: don't need both 're-' and 'again'.  So "...app UI and login to the BIG-IP again.

File: docs/faq.rst
Comment (JosephPJordan) [inline docs/faq.rst:111]: ... using a **ping** command.  (or 'the ping command')

File: docs/faq.rst
Comment (JosephPJordan) [inline docs/faq.rst:123]: The visibility VIP table...

File: docs/faq.rst
Comment (JosephPJordan) [inline docs/faq.rst:129]: ... to which the Virtual Server (VIP) or Node belongs.  
<don't need 'to' at the end with "to which" earlier in the sentence>

File: docs/faq.rst
Comment (JosephPJordan) [inline docs/faq.rst:133]: ...for the 'View Logs' window...

File: docs/faq.rst
Comment (JosephPJordan) [inline docs/faq.rst:135]: ...this issue, use the 'zoom out' option on your web browser. For example, on Windows, hold the **Ctrl** key, and then click **-** (the dash/minus key).

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:89]: ...VIP Dashboard, you must install the Telemetry Streaming plugin version 1.17 or higher on the BIG-IP device.

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:95]: For a scaled environment, we strongly recommend you retrieve just the Pool...

One way to filter the VIP... <missing the word 'to'>

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:133]: ...click the appropriate partition.

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:146]: The dashboard displays...

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:150]: The Telemetry Streaming...

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:154]: The Dashboard... link next to the default pool <or beside, not besides>
It displays the total pool member count...

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:156]: The Dashboard...  link next to the VIP, which, when clicked opens a popup window and displays...

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:164]: The Dashboard displays a **View Connections** link, which, when clicked displays all the active...

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:166]: The Visibility dashboard...

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:169]: The Visibility...

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:178]: ... click the appropriate partition.

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:188]: The Visibility Dashboard...

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:190]: The Dashboard displays.....  link next to the Node, which, when clicked opens a popup...

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:198]: The Dashboard displays.... which, when clicked displays all the active...

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:200]: The Visibility...

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:203]: The Visibility

File: docs/navigate.rst
Comment (JosephPJordan) [inline docs/navigate.rst:204]: "beside" or "next to"

File: docs/release-notes.rst
Comment (JosephPJordan) [inline docs/release-notes.rst:26]: ...using the following steps:

File: docs/release-notes.rst
Comment (JosephPJordan) [inline docs/release-notes.rst:77]: ..the Visibility tab

File: docs/release-notes.rst
Comment (JosephPJordan) [inline docs/release-notes.rst:120]: However, the configuration is removed from the BIG-IP device.

File: docs/release-notes.rst
Comment (JosephPJordan) [inline docs/release-notes.rst:133]: ..., the application is created on the BIG-IP.

File: docs/release-notes.rst
Comment (JosephPJordan) [inline docs/release-notes.rst:211]: If BIG-IP pool members are automatically added by the...

**Comment by JosephPJordan** [comment]
Hi @manuwelakanade-GSLAB, a few suggested changes.  Let me know if there's a better way we can do this (ie if i should just make changes directly to the files, etc).

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : Thanks for the review. I have addressed all your comments and updated the pull request. Can you please merge this request if there aren't any further comments. Once this is merged, I will raise a pull request from staging to master branch. Thank you.

**Review by JosephPJordan — APPROVED** [review]

**Comment by manuwelakanade-GSLAB** [comment]
Thanks @JosephPJordan

**Pull request merged**
